ast-nodes/expressions/operators/eq-operator.dart

```diff
@@ -53,7 +53,18 @@ class EqOperator extends BinaryRelation implements Comparison {
   }
 
   Pointer<LLVMOpaqueValue> generateCode(Module module) {
-    // TODO: implement
-    return null;
+    if (this.resultType is IntegerType) {
+      return llvm.LLVMConstICmp(
+        1, 
```

Comment:
It's best to actually use enum values, because it isn't clear from these numbers what they actually stand for. The enum is available as a global object `LLVMIntPredicate` with constants like `LLVMIntPredicate.LLVMIntNE`